mainmatter.md

```diff
@@ -309,19 +291,21 @@ This depends, however, on the AS to know what authorization details are relevant
 As an example, it is possible to specify that the client will get "read" access to "file X" stored at the resource "https://store.example.com". To achieve this, the example given above for access to an IMAP server is slightly modified to use the `resource` element as part of the top level claims within the `authorization_details` element. 
 
 ```JSON
-{  
-   "https://scheme.example.org/imap":{  
+[  
+   {
+      "type": "https://scheme.example.org/imap":
       "resource":"imap.example.com",
```

Comment:
I think resource and locations are mutual exclusive, we need to decide what element to use and how - see also #7

Reply:
I struggled with that, too. The thing is, resources can be logical names, and I kinda wanted location to be strictly URI's. So my thought was to leave them both in and let them be used side by side, for the moment, and then figure out what exactly the relationship and overlap is. 

You could make similar arguments about the PoP `aud` parameter, and likely a dozen proprietary extensions that do similar things. I don't think we can necessarily fold all of them in cleanly.